project.md

```diff
@@ -11,41 +11,59 @@ _What need is met by your idea? Who are you helping? What is that person's
+
 ### Expressiveness
 
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to store quantities and perform basic Economics formulas in google sheets such as: SUM() and VARIANCE(). This also means it should be possible to perform basic arithmetic operations (+, -, *, /) with the outputs of these formulas. People should be limited in what they can express to the scope of these operations and formulas from Google/Excel so that they are not confused when they combine different operations and see unclear behavior as they would not have the technical background to know how to resolve it or even recognize it as a bug at times.
+
+I am not fully certain of which finance formulas I want to implement, since I know I will have to limit them as there are many but I want to prioritize some of the most used ones such as: PV(), FV(), and NPV(). This will be an ongoing decision and depending on how much language design will go into each formula, I may do more or less to ensure I am not spending too much time per task relative to the functionality being gained.
+
 ### Related work
 
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+In researching other DSLs in this domain, I found it interesting to find that some people already consider Excel to a be a finance Domain Specific Language: https://medium.grid.is/excel-a-domain-specific-language-for-finance-e9ff6e6e3972#:~:text=Excel%20is%20indeed%20a%20Domain,darn%20good%20one%20at%20that. While I had not considered this, it does seem to be fulfill the definition of a DSL, being confined to a particular problem/domain and users do write in it to communicate with the computer which returns desired outputs. However, as I have discussed above, I am looking to improve upon this experience, though i will be lacking the useful UI it has. Additionally, I found that the python library numpy does contain some of the operations I want to implement: https://pythontic.com/finance/cashflow-modeling. Based off the documentation I looked over, it does look like it can be used to produce very readable code. There are some minor differences between the experience of using this library and my experience using Google sheets, including function names which I would like to maintain consistent but it seems generally similar to the goal I am aiming for with my DSL implementation.
```

Comment:
I would strongly encourage you to look into the numpy documentation and possibly consider the usage of python. I have previous experience with numpy and it has a ton of mathematical applications. This could make it significantly easier for your implementation process.